### PR TITLE
Double click

### PR DESCRIPTION
**Pull request by FBerthelot**

### CODE REVIEW

File: src/javascript/DesignSystem/TreeView/TreeView.jsx [hunk 1]
```diff
@@ -42,31 +42,44 @@ const style = theme => ({
...
                 }
+            };
 
+            const handleNodeClick = e => {
+                toogleNode();
                 onNodeClick(node, e);
             };
 
+            const handleNodeDoubleClick = e => {
+                toogleNode();
```
Comment (Taybou) [inline src/javascript/DesignSystem/TreeView/TreeView.jsx]:
```suggestion
                toggleNode();
```

File: src/javascript/DesignSystem/TreeView/TreeView.jsx [hunk 2]
```diff
@@ -42,31 +42,44 @@ const style = theme => ({
...
+            const toogleNode = () => {
                 if (nodeIsOpen) {
                     setOpenedNode(openedNodes.filter(n => n !== node));
                 } else {
                     setOpenedNode([...openedNodes, node]);
                 }
+            };
 
+            const handleNodeClick = e => {
+                toogleNode();
```
Comment (Taybou) [inline src/javascript/DesignSystem/TreeView/TreeView.jsx]:
```suggestion
                toggleNode();
```

File: src/javascript/DesignSystem/TreeView/TreeView.jsx [hunk 3]
```diff
@@ -42,31 +42,44 @@ const style = theme => ({
...
+const TreeViewCmp = ({tree, onNodeClick, onNodeDoubleClick, classes}) => {
     // By default everything is closed
     const [openedNodes, setOpenedNode] = useState([]);
 
     function generateLevelJSX(level, deep) {
         return level.map(node => {
             const nodeIsOpen = Boolean(openedNodes.includes(node));
 
-            const handleNodeClick = e => {
+            const toogleNode = () => {
```
Comment (Taybou) [inline src/javascript/DesignSystem/TreeView/TreeView.jsx]:
```suggestion
            const toggleNode = () => {
```

File: src/javascript/DesignSystem/TreeView/TreeView.jsx [hunk 4]
```diff
@@ -42,31 +42,44 @@ const style = theme => ({
...
+            };
+
             if (node.childs && node.childs.length !== 0) {
                 const Arrow = nodeIsOpen || node.opened ? ArrowDropDown : ArrowRight;
                 const Childs = nodeIsOpen || node.opened ? generateLevelJSX(node.childs, deep + 1) : <></>;
 
                 return (
                     <Fragment key={level.id + node.id}>
-                        <button type="button" className={classes.nodeWithChild} onClick={handleNodeClick}>
+                        <button
```
Comment (Taybou) [inline src/javascript/DesignSystem/TreeView/TreeView.jsx]: Why we don't use Button component ?

Comment (FBerthelot) [reply src/javascript/DesignSystem/TreeView/TreeView.jsx]: Because it doesn't look like our Button Component

Comment (Taybou) [reply src/javascript/DesignSystem/TreeView/TreeView.jsx]: okay hhh

**Pull request merged**